tests/docker/Dockerfile

```diff
@@ -76,7 +77,7 @@ RUN apt-get install fuse
 RUN cd /opt && git clone -q  https://github.com/confluentinc/kibosh.git && cd "/opt/kibosh" && git reset --hard $KIBOSH_VERSION && mkdir "/opt/kibosh/build" && cd "/opt/kibosh/build" && ../configure && make -j 2
 
 # Set up the ducker user.
-RUN useradd -ms /bin/bash ducker && mkdir -p /home/ducker/ && rsync -aiq /root/.ssh/ /home/ducker/.ssh && chown -R ducker /home/ducker/ /mnt/ /var/log/ && echo 'ducker ALL=(ALL) NOPASSWD: ALL' >> /etc/sudoers
+RUN useradd -ms /bin/bash ducker && mkdir -p /home/ducker/ && rsync -aiq /root/.ssh/ /home/ducker/.ssh && chown -R ducker /home/ducker/ /mnt/ /var/log/ && echo "PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:$JAVA_HOME/bin" >> /home/ducker/.ssh/environment && echo 'ducker ALL=(ALL) NOPASSWD: ALL' >> /etc/sudoers
```

Comment:
We should append elements to PATH, not replace it completely.  Like PATH="$PATH:<whatever>"

Reply:
I wasn't sure if that would work with .ssh/environment, but I should confirm that.

Reply:
Indeed, expansion does not work in .ssh/environment. It only replaces.

Reply:
I've changed this now so that it user runuser to get ducker user's `$PATH` and add `$JAVA_HOME/bin` to it.